apps/s_server.c

```diff
@@ -3000,7 +3000,9 @@ static int www_body(int s, int stype, int prot, unsigned char *context)
     /* Set width for a select call if needed */
     width = s + 1;
 
-    p = buf = app_malloc(bufsize, "server www buffer");
+    /* as we use BIO_gets(), and it always null terminates data, we need
+     * to allocate 1 byte longer buffer to fit the full 2^14 byte record */
+    p = buf = app_malloc(bufsize + 1, "server www buffer");
```

Comment:
Wouldn't it be better to just change the definition of `bufsize` to be the *real* buffer size?

Reply:
not given the other places its used in, the www and rev is the exception not the rule in s_server

Reply:
ok